app/controllers/attendances_controller.rb

```diff
@@ -64,8 +74,16 @@ def set_attendance
       @attendance = Attendance.find(params[:id])
     end
 
+    def set_employee
+      @employee = Employee.find_by private_number: params[:private_number]
+      unless @employee.present?
+        return render json: {"message": "The user with these private number does not exist"}, status: :unprocessable_entity
+      end
+    end
+
     # Only allow a list of trusted parameters through.
     def attendance_params
       params.require(:attendance).permit(:check_in, :check_out, :employee_id, :user_id)
     end
+
```

Comment:
Please remove these unnecessary extra line

Reply:
i think this extra line at the end of every file is added by default with an extension, someone mentioned before, it was a good practice add it, it isn't?.